### ISSUE TITLE
[Bug Report] toast

### ISSUE THREAD

**Issue by 1393254712** [youzan/vant-weapp#5754]
### 重现链接

Toast.clear();
Toast({message: '无更多数据'});

### Vant Weapp 版本

1.11.4

### 描述一下你遇到的问题。

clear后 在后面的Toast也会被清除

### 重现步骤

Toast.clear();
Toast({message: '无更多数据'});

**Comment by johnsonwong666** [comment]
Toast({message: '无更多数据'});
这里加个setTimeout包一下 delay 0

**Comment by 1393254712** [comment]
> Toast({message: '无更多数据'}); 这里加个setTimeout包一下 delay 0

0还是会闪现消失

**Comment by johnsonwong666** [comment]
可以简单提供一个代码片段吗？

**Comment by johnsonwong666** [comment]
https://developers.weixin.qq.com/s/FMLMEgmJ7CQJ

在这上面改一下 然后分享回来

**Comment by johnsonwong666** [comment]
https://developers.weixin.qq.com/s/FMLMEgmJ7CQJ

在这上面改一下 然后分享回来

**Comment by 1393254712** [comment]
> 可以简单提供一个代码片段吗？

`classHomeworkApi.getClassHomeworkList(account.id, params , res => {
        Toast.clear();
        if (res.status == ResultCodeConstant.SUCCESS_CODE) {
        } 
        else {
         Toast({message: res.message});
        }
      }, err => {
        Toast.clear();
        Toast({message: err.message});
      });`

**Comment by johnsonwong666** [comment]
什么原因需要加一个Toast.clear()呢？  感觉这个是不是多余了

**Comment by 1393254712** [comment]
> 什么原因需要加一个Toast.clear()呢？ 感觉这个是不是多余了

因为我有的地方只需要清除toast 不需要显示别的提示
不然就只能在不显示toast的地方单独clear一下 才能避免问题

**Comment by johnsonwong666** [comment]
这里先clear 紧接着Toast别的内容，这里代码应该存在一点问题。按照你的情况的话 在catch里面 直接Toast别的内容就可以了

**Comment by 1393254712** [comment]
> 这里先clear 紧接着Toast别的内容，这里代码应该存在一点问题。按照你的情况的话 在catch里面 直接Toast别的内容就可以了

是的 这个确实没必要加 我在res里面有几个if 有一个需要Toast一下， 所以在进入res里面就直接clear了 我如果快速请求两次 触发了两次clear 就会导致后面的toast给清除了

**Comment by 1393254712** [comment]
> 这里先clear 紧接着Toast别的内容，这里代码应该存在一点问题。按照你的情况的话 在catch里面 直接Toast别的内容就可以了

是的 这个确实没必要加 我在res里面有几个if 有一个需要Toast一下， 所以在进入res里面就直接clear了
